packages/build-config/sections/module-rules/postcss.js

```diff
@@ -30,7 +30,9 @@ exports.build = {
     use: [
       {
         loader: require.resolve('css-loader'),
-        options: cssLoaderOptions,
+        options: Object.assign({}, cssLoaderOptions, {
+          localIdentName: '[hash:base64:8]',
```

Comment:
We will also have to do this for node to make sure that the generated markup is equal no matter where an in what env it is rendered.